Adds fix for background image CSS breaking

coreymcollins opened the pull request:
### DESCRIPTION
@lindseywb and @khleomix noticed a problem where setting a background image in a Sass partial was causing an error when building:
<img width="1135" alt="image" src="https://user-images.githubusercontent.com/954724/129785136-1191aa91-fe9f-4a30-92f4-9ae2d4d73177.png">

Adding a file loader to our Webpack config seems to solve this – I can now set a background image in Sass or in the Tailwind config file as a theme extend and use that class without error.

```
{
    test: /\.(png|jpe?g|gif)$/,
    include: /images/,
    use: [
        {
            loader: 'file-loader',
            options: {
                name: '[name].[ext]',
                outputPath: 'images/',
                publicPath: 'images/',
            },
        },
    ],
},
```

### OTHER

- [x] Is this issue accessible? (Section 508/WCAG 2.0AA)
- [x] Does this issue pass all the linting? (PHPCS, ESLint, SassLint)
- [x] Does this pass CBT?

### STEPS TO VERIFY

1. Checkout `main` and add a background image to something in a Sass partial + run `npm run build` to get the error
2. Checkout this branch
3. Add a background image to something in a Sass partial + run `npm run build` to fix the error and have the background image CSS process without issue

### DOCUMENTATION

Will this pull request require updating the wd_s [wiki](https://github.com/WebDevStudios/wd_s/wiki)?

Don't think this requires any documentation updates.

coreymcollins committed:
Adds fix for background image CSS breaking

gregrickaby commented on webpack.config.js:
```diff
@@ -16,6 +16,20 @@ module.exports = {
 	module: {
 		rules: [
+			{
+				test: /\.(png|jpe?g|gif)$/,
```
@coreymcollins Looks good, but I think we should consider adding support for `.webp` and maybe `.svg`?

coreymcollins replied:
Sure, I had initially not included SVG since we have those in their own icons directory but I could see a case where we have an SVG living outside of that directory.

coreymcollins committed:
Adds file-loader support for .webp and .svg

coreymcollins commented:
Added support for `.webp` and `.svg` – had to change the `include` line a little bit as well, because it seemed to be grabbing _some_ images from inside of the `/images/icons/` directory once we started listening for `.svg` files.

coreymcollins committed:
Updates file loader to resolve a breaking issue

coreymcollins commented:
I've pushed another update here to fix a weird issue we were hitting.

I updated this:
```
{
    test: /\.(svg|webp|png|jpe?g|gif)$/,
    include: '/images/[name].[ext]',
    use: [
        {
            loader: 'file-loader',
            options: {
                name: '[name].[ext]',
                outputPath: 'images/',
                publicPath: 'images/',
            },
        },
    ],
},
```

To this:
```
{
    test: /\.(webp|png|jpe?g|gif)$/,
    use: [
        {
            loader: 'file-loader',
            options: {
                name: '[name].[ext]',
                outputPath: 'images/',
                publicPath: 'images/',
            },
        },
    ],
},
```

## Why We Did It
We noticed that we were getting errors when building, as seen in the first screenshot in the initial post.

## What We Did
I removed `svg` from the file list. Oddly, this still allows background image SVGs to work as they are still copied to the `/images/` directory via our other Webpack actions.

I also removed the `/include` which seemed to cause problems as well, including (but not limited to) copying _some_ files from `/images/icons/` into the top-level `/build/images/` directory.

## What It Does
Everything still works the same. Anything in `/src/images/` is still copied to `/build/images/` and we get no errors, plus no weird copying of `/images/icons/` files into `/images/`.

Additionally, we're now able to use Tailwind to extend the theme to setup background images and/or use `background`/`background-image` in our partials to manually set background images. Both of these were failing before because there was no `file-loader` in place.

## How I Tested
Doing all of this seems to work for me and compiles the backgrounds as expected.

`webpack.config.js`
```{
    test: /\.(webp|png|jpe?g|gif)$/,
    use: [
        {
            loader: 'file-loader',
            options: {
                name: '[name].[ext]',
                outputPath: 'images/',
                publicPath: 'images/',
            },
        },
    ],
},
```

`tailwind.config.js`
```backgroundImage: {
    'test-one': "url('/build/images/facebook-square.svg')",
    'test-two': "url('/build/images/placeholder.png')",
    'test-three': "url('/build/images/bull.jpeg')",
},
```

`_header.scss`
```//--------------------------------------------------------------
// Global Site Header
//--------------------------------------------------------------
.site-header {
	@apply bg-black bg-opacity-10 py-12 bg-test-three;

	> .container {
		@apply flex justify-between my-0;
	}

	.site-description {
		@apply no-underline bg-test-one;
	}

	.site-branding {
		a {
			@apply no-underline bg-test-two;
		}
	}
}
```

Worth another round of testing!

coreymcollins committed:
Squashed commit of the following:

Mergers main in

coreymcollins committed:
Replaces file-loader with asset modules

coreymcollins committed:
Simplifies filename output for images
